src/server/middleware/security/oidc_cli.go

```diff
@@ -65,6 +65,11 @@ func (o *oidcCli) Generate(req *http.Request) security.Context {
 
 	info, err := oidc.VerifySecret(ctx, username, secret)
 	if err != nil {
+		user, err2 := uctl.GetByName(ctx, username)
+		// skip to log the error if the user is the admin user
+		if err2 == nil && user != nil && user.UserID == 1 {
+			return nil
```

Comment:
Log at debug level for admin users as this might be expected

logger.Debugf("OIDC verification failed for admin user %s: %v", username, err)